pydoctor/__init__.py

```diff
@@ -4,10 +4,17 @@
 
 """
 
-__all__ = ["__version__"]
+from typing import TYPE_CHECKING
+
+# On Python 3.8+, use importlib.metadata from the standard library.
+# On older versions, a compatibility package can be installed from PyPI.
+try:
+    import importlib.metadata as importlib_metadata
+except ImportError:
+    if not TYPE_CHECKING:
+        import importlib_metadata
 
-from ._version import __version__
 
+__version__ = importlib_metadata.version('pydoctor')
```

Comment:
changing this from an Incremental instance to a str is a breaking change - I think you might as well forgo the `__version__ =` entirely and require callers `importlib.metadata.version("pydoctor")` if they ever forget what version they installed

Reply:
That's a fair point, but pydoctor doesn't offer a stable API yet, so there shouldn't be many callers. Even Twisted's customizations could be phased out; see #315.